docs/fast_data/tutorials/low-code.mdx

```diff
@@ -26,348 +26,347 @@ To do that, in the console, you will:
 
 ## Prerequisites
 
-Before starting this tutorial, you need to be familiar with the concepts of [Fast Data architecture](/fast_data/architecture.md), including [Projections](/fast_data/configuration/projections.md), [Single View](/fast_data/configuration/single_views.md), and Kafka (including producers and consumers).
-For the technical part of this tutorial, an up-and-running Kafka cluster is required.
+Before starting this tutorial, you need to be familiar with the concepts of:
+* basic knowledge about [Fast Data architecture](/fast_data/architecture.md), in particular:
+  * [Standard Architecture](/fast_data/architecture.md#standard-architecture)
+  * [Event-Driven Architecture](/fast_data/architecture.md#event-driven-architecture)
+* familiarity with the concepts of: 
+  * [Projections](/fast_data/configuration/projections.md)
+  * [Single View](/fast_data/configuration/single_views.md)
+  * [Strategies](/fast_data/configuration/strategies.md)
+  
+* Kafka (including producers and consumers).
+* MongoDB
 
-## System of Record
+For the technical part of this tutorial, you will need both Kafka and MongoDB instances available.
 
-A System of Record is a container of _Projections_, which is are architectural objects responsible for storing the data in a persistent storage, to have a fast and scalable way to decouple the data from the source system.
-We will use these projections to have an easy way to read a 1:1 copy of the data from the source systems, and later they will be used to aggregate the data from the source systems, using the Single View Creator.
+## Systems of Record
 
-The architectural component responsible for creating and updating the projection is the _Real-Time Updater_ service.
-To simplify, this microservice is a Kafka consumer that will consume the data from the source systems and will update the projection.
+A _System of Record_ (__SoR__) is a container of _Projections_, which is are architectural objects responsible for storing the data in a persistent storage, to have a fast and scalable way to decouple the data from the source system.
+We will use these projections to have a copy of the data from the source systems, and later they will be used to aggregate the data from the source systems, using the Single View Creator.
+
+The architectural component responsible for creating and updating the projection is either the _Real-Time Updater_ or, starting from `Console v12.0.0`, the _Projection Storer_: we will refer to it as `SoR Microservice` throughout this tutorial.
+
```

Comment:
Maybe I would add a callout to underline that the Projection Storer service is only available from Console >= 12.0.0